quickstart.md

```diff
@@ -174,13 +174,85 @@ root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/contain
 
 ## Creating a CoCo workload using a pre-existing encrypted image
 
-We will now proceed to download and run an encrypted container image using the CoCo building blocks.
+We will now proceed to download and run a sample encrypted container image using the CoCo building blocks.
 
-* *TBD: based on https://github.com/confidential-containers/operator/issues/77* *
+A demo container image is provided at [docker.io/katadocker/ccv0-ssh](https://hub.docker.com/r/katadocker/ccv0-ssh).
+It is encrypted with [Attestation Agent](https://github.com/confidential-containers/attestation-agent)'s [offline file system key broker](https://github.com/confidential-containers/attestation-agent/tree/64c12fbecfe90ba974d5fe4896bf997308df298d/src/kbc_modules/offline_fs_kbc) and [`aa-offline_fs_kbc-keys.json`](./aa-offline_fs_kbc-keys.json) as its key file.
```

Comment:
I think it should like to https://github.com/confidential-containers/documentation/blob/main/demos/ssh-demo/aa-offline_fs_kbc-keys.json rather than `./aa-offline_fs_kbc-keys.json`.

Other than that, this PR is ready to go. Thanks again @stevenhorsman !